scripts/getPVAliases.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/bash
+
+# define usage
+usage()
+{
+cat << EOF
+usage: gatherPVAliases [-h] [-d] patt hutch
+
+gathers all record <-> alias associations from a child's ioc.cfg, st.cmd, and
+parent ioc.cfg.
+
+positional arguments:
+  patt
+  hutch
+
+options:
+  -h, --help     show this help message and exit
+  -d, --dry_run  Forces a dry run for the script. No files are saved.       
+
+EOF
+}
+
+# catch improper number of args or asking for help
+
+if [[ ("$#" -lt 2) || ("$1" == "-h") || ("$1" == "--help") ]]; then
+    usage
+    exit 0
+fi
+
+# source into pcds_conda if not currently active
```

Comment:
Were you running into issues sourcing `pcds_conda` after it's been already activated?  I've never had an issue doing so

Reply:
It really came down to saving 2-3 seconds per execution in a different script that I based a lot of my ioc manipulation on. That script called `pandas` which is a notoriously long import so I was trying to shave time off as much as I could.